components/interactive_dialog/dialog_introduction_text.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import * as Markdown from 'utils/markdown';
+import {getSiteURL} from 'utils/url';
+
```

Comment:
nit : no empty line

Reply:
@M-ZubairAhmed To be fair these empty line changes are a bit annoying, please add lint rules as soon as possible